### PR TITLE
Opera 70 released

### PR DESCRIPTION
**Pull request by afmenez**
Opera 70 released this week.  I am also adding versions 68 and 69, which were missing.  Finally, adding release dates for all versions.

### CODE REVIEW

File: environments.json [hunk 1]
```diff
@@ -2603,370 +2603,431 @@
...
     "short": "OP 58",
-    "obsolete": "very"
-  },
-  "opera59": {
-    "full": "Opera 59",
-    "family": "V8",
-    "equals": "chrome72",
-    "short": "OP 59",
+    "release": "2019-01-23",
     "obsolete": "very"
```
Comment (ljharb) [inline environments.json:2951]: what happened to opera 59?

File: environments.json [hunk 2]
```diff
@@ -2603,370 +2603,431 @@
...
+    "release": "2019-04-09",
     "obsolete": "very"
   },
-  "opera61": {
-    "full": "Opera 61",
-    "family": "V8",
-    "equals": "chrome74",
-    "short": "OP 61",
-    "obsolete": true
-  },
```
Comment (ljharb) [inline environments.json:2929]: and opera 61?

File: environments.json (hunk 1 above)
Comment (afmenez) [reply environments.json:2951]: According to my [source](https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser#Opera_2019), version 59 was skipped.

File: environments.json (hunk 2 above)
Comment (afmenez) [reply environments.json:2929]: According to my [source](https://en.wikipedia.org/wiki/History_of_the_Opera_web_browser#Opera_2019), version 61 was skipped.

**Review by ljharb — APPROVED** [review]

File: environments.json [hunk 3]
```diff
@@ -2603,370 +2603,431 @@
...
+    "obsolete": "very"
   },
   "opera63": {
     "full": "Opera 63",
     "family": "V8",
     "equals": "chrome76",
     "short": "OP 63",
-    "obsolete": true
+    "release": "2019-08-20",
+    "obsolete": "very"
```
Comment (chicoxyzzy) [inline environments.json]: I think this one should be `true` until August 2020

**Review by chicoxyzzy — APPROVED** [review]

File: environments.json (hunk 3 above)
Comment (afmenez) [reply environments.json]: Done, but I believe we should be trying to reduce, not increase, the number of the columns in the table.

**Comment by chicoxyzzy** [comment]
thanks!

**Pull request merged**